framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1064,6 +1064,12 @@ public AnnotationMirror getAnnotationMirror(Tree tree, Class<? extends Annotatio
      * @return the annotated type of {@code tree}
      */
     public AnnotatedTypeMirror getAnnotatedType(Tree tree) {
+
+        String treeString = tree.toString();
```

Comment:
This seems to be left-over from some debugging. The `treeString` variable isn't used and computing it can be costly.